modules/op-specifying-pipelines-resource-quota-using-priority-class.adoc

```diff
@@ -130,50 +130,49 @@ spec:
 .Example: YAML for a task in the pipeline
 [source,yaml]
 ----
-apiVersion: tekton.dev/v1alpha1
+apiVersion: tekton.dev/v1beta1
 kind: Task
 metadata:
   name: mvn
 spec:
   workspaces:
   - name: maven-repo
-  inputs:
-    params:
-    - name: GOALS
-      description: The Maven goals to run
-      type: array
-      default: ["package"]
-    resources:
+  resources:
+    inputs:
     - name: source
       type: git
+  params:
+  - name: GOALS
+    description: The Maven goals to run
+    type: array
+    default: ["package"]
   steps:
     - name: mvn
       image: gcr.io/cloud-builders/mvn
       workingDir: /workspace/source
       command: ["/usr/bin/mvn"]
       args:
         - -Dmaven.repo.local=$(workspaces.maven-repo.path)
-        - "$(inputs.params.GOALS)"
-      priorityClassName: pipeline1-pc
+        - "$(params.GOALS)"
 ----
-+
-[NOTE]
-====
-Ensure that all tasks in the pipeline belongs to the same priority class.
-====
```

Comment:
@concaf @vdemeester @ppitonak I removed this note since we do not specify the `priorityClassName` field in the tasks, but in the pipeline run. So, this note no longer made sense to me.